syntax = "proto2";
package com.my.algorithm.proto;

option optimize_for = LITE_RUNTIME;
//option cc_enable_arenas = false;

//// For Java.
//option java_outer_classname = "ProtoGridList";

import "protocol/ProtoGrid.proto";

message MsgGridList {
    repeated MsgGrid items = 1;
}
